src/v2/cookbook/using-axios-to-consume-apis.md

```diff
@@ -159,21 +160,21 @@ new Vue({
 </div>
 ```
 
-You can hit the rerun button on this pen to see the loading status briefly while we gather data from the API:
+Puedes accionar el botón rerun en esta nota para ver el estado de cargando brevemente, mientras se solicitan los datos al API:
```

Comment:
It's perfect, just change it to use the formal language type.

For example:

use "Puede accionar" instead of "Puedes accionar"
use "Puede que haya escuchado" instead of "Puede que hayas escuchado"

Reply:
:+1: I'll take a look tomorrow, thanks